ppcs/ppc0013-overload-join-and-substr.md

```diff
@@ -98,6 +98,8 @@ We do not yet foresee security issues. Guidance is welcome.
 For `join`:
 
 ```perl
+use List::Util;
```

Comment:
reduce must be imported explicitly from List::Util, there are no default exports.

Reply:
Thanks will rework

Reply:
Done @Grinnz thanks for the feedback!